compiler/x/codegen/X86Ops.hpp

```diff
@@ -1217,6 +1217,99 @@ typedef enum
    IA32EFlags_CF = 0x10
    } TR_EFlags;
 
+namespace TR
```

Comment:
My apologies for the delay in responding, but I've been giving this some deeper thought on how to proceed and I have a different suggestion.  There's been some discussion around the use of the `TR` namespace (issue #788), and I have to admit I'm more convinced of its use as a pure namespace without further refinement.

Looking at what you've implemented, why don't you implement these as public static members of the `OMR::X86::Instruction` class?  Then, you would simply reference them as `TR::Instruction::ModRM` (for example).  The way you reference things in the `TR` namespace now makes the developer think they are static members, but they're not.  Making them static members is cleaner and conceptually pleasing.